pkg/codegen/docs/templates/index_categories.tmpl

```diff
@@ -2,7 +2,7 @@
 <h2 id="modules">Modules</h2>
 <ul class="api">
 {{- range . }}
-    <li><a href="{{ .Link }}" title="{{ .DisplayName }}"><span class="api-symbol api-symbol--module"></span>{{ .DisplayName }}</a></li>
+    <li><a href="{{ .Link }}" title="{{ .DisplayName }}">{{ .DisplayName }}</a></li>
```

Comment:
```suggestion
    <li><a href="{{- .Link -}}" title="{{- .DisplayName -}}">{{ .DisplayName }}</a></li>
```

Reply:
what does that do?

Reply:
It strips whitespace on either side (`{{-` strips whitespace on the left, `-}}` on the right. Not a big deal; I guess it's most useful in the href since that's the most whitespace sensitive, I think?